test/models/report_test.rb

```diff
@@ -3,7 +3,38 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test '#editable?' do
+    alice_report = reports(:alice_report)
+
+    assert alice_report.editable?(users(:alice))
+    assert_not alice_report.editable?(users(:bob))
+  end
+
+  test '#created_on' do
+    alice = users(:alice)
+    report = alice.reports.create!(title: 'alice_report', content: '自分が聴いだのは近頃途中でおもにないうでし。', created_at: '2024-02-27 21:00'.in_time_zone)
```

Comment:
```suggestion
    report = reports(:alice_report)
    report.created_at = '2024-02-27 21:00'.in_time_zone
```

注目したい点はcreated_atとcreated_onの関係だけなのに、それ以外のコードがたくさんあるとノイズになって何をテストしたいのかわかりづらくなります。
なのでこんなふうに必要最小限のテストコードにする方がベターです。

Reply:
ありがとうございます! 修正しました。